code/game/machinery/flasher.dm

```diff
@@ -87,6 +87,13 @@
 	if (anchored)
 		return flash()
 
+/obj/machinery/flasher/eminence_act(mob/living/simple_animal/eminence/eminence)
+	. = ..()
+	to_chat(usr, "<span class='brass'>You begin manipulating [src]!</span>")
+	if(do_after(eminence, 20, target=get_turf(eminence)))
+		if(anchored)
+			flash()
```

Comment:
Why don't return it early with some message if not anchored?

Reply:
need to check again because it can be unanchored during the time